### ISSUE TITLE
Make default busy-wait delay much shorter

### ISSUE THREAD

**Issue by birkenfeld** [thejpster/embedded-sdmmc-rs#1]
Using a sdcard with a low-frequency STM32L0, the default 100000 spins was much too long, I reduced it to 100. (With that, everything works perfectly, thanks very much for the library!)

IMO if we're busy-waiting anyway, there shouldn't be a downside to polling more often also on faster CPUs?

**Comment by thejpster** [comment]
> /// A terrible hack for busy-waiting the CPU while we wait for the card to
> /// sort itself out.
> ///
> /// @TODO replace this!

Yes, it's terrible! I'll try 100 on my board and if that works, push the change.

Glad to know it works on an STM32 - that's a great indication that Rust Embedded's HAL approach works! If you can, please do tweet @rust_embedded and let them know :)

**Comment by thejpster** [comment]
PS: The delay was only there because I was worried that hammering the card as fast as we good made one of the cards I was testing with a bit upset.

**Comment by birkenfeld** [comment]
That's perfect,thank you!
